runtime/standalone/src/lib.rs

```diff
@@ -286,7 +286,7 @@ pub struct FixedGasPrice;
 impl FeeCalculator for FixedGasPrice {
 	fn min_gas_price() -> U256 {
 		// Gas price is always one token per gas.
-		1.into()
+		0.into()
```

Comment:
Is this intentional? If so, maybe part of a separate PR.

Reply:
Yes, to keep it same as parachain. No need for a specific PR on this one I believe as it is part of the effort of merging